client/package.json

```diff
@@ -20,7 +20,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "storybook": "start-storybook -p 6006 -s public",
+    "storybook": "start-storybook -p 6006 --no-manager-cache -s public",
```

Comment:
Why do we need no-manager-cache flag here?  Maybe it's better to put it in storybook config and leave comment.

Reply:
I removed this flag from scripts. It can severely impact Storybook loading time: https://storybook.js.org/docs/react/api/cli-options